<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPlainTextEdit" name="plainTextEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>878</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menuFont_Size">
     <property name="title">
      <string>Font Size</string>
     </property>
     <addaction name="action6_px"/>
     <addaction name="action11_px"/>
     <addaction name="action12_px"/>
     <addaction name="action14_px"/>
     <addaction name="action16_px"/>
     <addaction name="action24_px"/>
     <addaction name="action30_px"/>
     <addaction name="action60_px"/>
     <addaction name="action72_px"/>
     <addaction name="separator"/>
    </widget>
    <addaction name="menuFont_Size"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="action6_px">
   <property name="text">
    <string>6 px</string>
   </property>
  </action>
  <action name="action11_px">
   <property name="text">
    <string>11 px</string>
   </property>
  </action>
  <action name="action12_px">
   <property name="text">
    <string>12 px</string>
   </property>
  </action>
  <action name="action14_px">
   <property name="text">
    <string>14 px</string>
   </property>
  </action>
  <action name="action16_px">
   <property name="text">
    <string>16 px</string>
   </property>
  </action>
  <action name="action24_px">
   <property name="text">
    <string>24 px</string>
   </property>
  </action>
  <action name="action30_px">
   <property name="text">
    <string>30 px</string>
   </property>
  </action>
  <action name="action60_px">
   <property name="text">
    <string>60 px</string>
   </property>
  </action>
  <action name="action72_px">
   <property name="text">
    <string>72 px</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
